torch/utils/_sympy/value_ranges.py

```diff
@@ -105,9 +112,13 @@ def __or__(self, other) -> "ValueRanges":
             return ValueRanges.unknown()
         assert self.is_bool == other.is_bool, (self, other)
         if self.is_bool:
-            range = ValueRanges(sympy.And(self.lower, other.lower), sympy.Or(self.upper, other.upper))
+            range = ValueRanges(
+                sympy.And(self.lower, other.lower), sympy.Or(self.upper, other.upper)
```

Comment:
These are all formatting correct?

Reply:
Fixed this PR to exclude formatting changes.